### ISSUE TITLE
Review my Resume

### ISSUE THREAD

**Issue by AgnelSelvan** [EddieHubCommunity/support#1026]
**Resume**

- [Resume](https://drive.google.com/file/d/1CewaN4MLZd6rWt03GrmbdP7TeGWKc5qD/view?usp=sharing)

Extra Links
- [Github Profile](https://github.com/AgnelSelvan)
- [Portfolio](http://agnelselvan.me/)

You can also join the discord community [here](https://discord.com/invite/jZQs6Wu)

Feel free to check out other cool repositories of EddieHub Community [here](https://github.com/EddieHubCommunity)

**Comment by adhavalboy** [comment]
change front and this can be possible to fit In one page

**Comment by DevCassie** [comment]
Things I would suggest looking into:
* Skill bars aren't aligned properly on the left and the right.
![image](https://user-images.githubusercontent.com/19227530/109543713-edffa700-7ac6-11eb-8d9c-ed0a14c45e3d.png)
* This is very cramped on each other, perhaps a bit of more space looks a bit more interesting.
![image](https://user-images.githubusercontent.com/19227530/109543730-f3f58800-7ac6-11eb-895c-a349ae7aaba6.png)
* Inconsistent use of margin/padding
![image](https://user-images.githubusercontent.com/19227530/109543928-3a4ae700-7ac7-11eb-8d87-4690db55b3be.png)

**Comment by AgnelSelvan** [comment]
Yeah, Thank you for your wonderful feedback. sure I'll be fixing this adjustment, Fonts.

**Comment by eddiejaoude** [comment]
Thanks for sharing 👍 I will review on a live stream soon. It looks like you are getting great feedback from the community already 🎉

**Comment by AgnelSelvan** [comment]
Thank you @eddiejaoude

**Comment by schmelto** [comment]
love the portfolio:

maybe take a look at this issue
![image](https://user-images.githubusercontent.com/30869493/117696492-a5005900-b1c1-11eb-8f8b-be18d3e1ee54.png)

**Comment by eddiejaoude** [comment]
Reviewed on live stream https://youtu.be/biy__J7abMU

Looks good, some suggestions
- Suggest using https (not a big deal though)
- Yellow text on white background is difficult to read (not accessible)
- percentages on technologies is not a great metric - also 50% is higher than %65

**Comment by Rishit-dagli** [comment]
None of the project links are clickable
